BlogAPI.Persistence/EntityFrameworkCore/Configurations/BlogPostConfiguration.cs

```diff
@@ -0,0 +1,41 @@
+﻿using BlogAPI.Persistence.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace BlogAPI.Persistence.Configurations
+{
+    internal class BlogPostConfiguration : IEntityTypeConfiguration<BlogPost>
+    {
+        public void Configure(EntityTypeBuilder<BlogPost> builder)
+        {
+            builder.HasKey(c => c.Id);
+            builder.Property(c => c.Id).ValueGeneratedOnAdd();
+            builder.Property(c =>  c.Title)
+                .IsRequired()
+                .HasMaxLength(50);
+            builder.Property(c => c.Author)
+                .IsRequired()
+                .HasMaxLength(50);
+            builder.Property(c => c.Content)
+                .IsRequired()
+                .HasMaxLength(5000);
+            builder.Property(c => c.CreationDate)
+                .IsRequired();
+            builder.Property(c => c.Tags)
+                .HasConversion(
+                    v => JsonSerializer.Serialize(v, (JsonSerializerOptions)null),
+                    v => JsonSerializer.Deserialize<List<string>>(v, (JsonSerializerOptions)null),
```

Comment:
Probably need spaces here? `v => JsonSerializer.Serialize(v, (JsonSerializerOptions)null)` -> `v => JsonSerializer.Serialize(v, (JsonSerializerOptions) null)` (you should really look into setting up some sort of a linter so it would format all of these minor things randomly)

Reply:
The recommended Microsoft whitespace style is to have no space after types in casts
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/code-style-rule-options
> csharp_space_after_cast = false